robottelo/entities.py

```diff
@@ -1616,13 +1637,12 @@ class Product(
         orm.Entity, orm.EntityReadMixin, orm.EntityDeleteMixin,
         orm.EntityCreateMixin):
     """A representation of a Product entity."""
-    organization = entity_fields.OneToOneField('Organization', required=True)
-    location = entity_fields.OneToOneField('Location', null=True)
```

Comment:
`location` is not mentioned in the API docs on either upstream or downstream systems.